src/interpreter/kernel/instruction/unop.js

```diff
@@ -19,6 +19,18 @@ function unop(
 
     case "neg":
       return createValue(c.neg());
+
+    case "clz":
+      return createValue(c.clz());
+
+    case "ctz":
+      return createValue(c.ctz());
+
+    case "popcnt":
+      return createValue(c.popcnt());
+
+    case "eqz":
+      return createValue(c.eqz());
```

Comment:
By the way, why is the value called `c`? value sounds more explicit to me.

Reply:
good point - I'll change that.